src/DoctrineBridge/Router/EntityIdResolver.php

```diff
@@ -25,11 +25,18 @@
     private $manager;
 
     /**
-     * @param ManagerRegistry $manager
+     * @var IdFromEntityInterface
```

Comment:
Actually, this docblock is not needed at all as IDEs can infer the type from the constructor.